README.mediawiki

```diff
@@ -514,6 +514,9 @@ The following flavors of Linux are supported
 
 == Changes ==
 
+;2.0.6
+* Fix LinuxMonitor - util.py lv_attributes references invalid position (ZEN-25792)
```

Comment:
The change should be documented in a way that's relevant to the user. Something like..

> Fix "string index out of range" error when modeling older LVM versions (ZEN-25792)

The idea is that a user might be able to decide if upgrading to a newer version would be useful to them.